components/submenu_modal/submenu_modal.test.jsx

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import {Modal} from 'react-bootstrap';
+
+import {browserHistory} from 'utils/browser_history';
+
+import SubMenuModal from 'components/submenu_modal/submenu_modal.jsx';
+
+describe('components/submenu_modal', () => {
+    const baseProps = {
+        elements: [
+            {
+                id: 'A',
+                text: 'A',
+            },
+            {
+                id: 'B',
+                text: 'B',
+                subMenu: [
+                    {
+                        id: 'C',
+                        text: 'C',
```

Comment:
nit: please have `id` and `text` with different values, maybe `{id: 'A', text: 'Text A'}`, just to exactly determine which one was called when asserted with `toHaveBeenCalledWith`

Reply:
Great idea, added.